Dockerfiles/Dockerfile.smoke_test

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.10.3
+
+RUN openssl genrsa -out /var/local/private.pem 2048
+RUN openssl rsa -in /var/local/private.pem -outform PEM -pubout -out /var/local/public.pem
+
+RUN go get -u github.com/golang/dep/cmd/dep
+
+WORKDIR /go/src/github.com/CMSgov/bcda-app
+COPY . .
+
+RUN dep ensure
+
+ENV BCDA_ERROR_LOG /var/log/bcda-error.log
```

Comment:
Again, don't think the client needs any of these `ENV` vars defined. Can probably remove.

Reply:
Removed.